src/Microsoft.ML.StandardLearners/Standard/Online/LinearSvm.cs

```diff
@@ -69,7 +69,8 @@ public sealed class Options : OnlineLinearOptions
             /// Column to use for example weight.
             /// </summary>
             [Argument(ArgumentType.AtMostOnce, HelpText = "Column to use for example weight", ShortName = "weight", SortOrder = 4, Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly)]
-            public Optional<string> WeightColumn = Optional<string>.Implicit(DefaultColumnNames.Weight);
+            [BestFriend]
```

Comment:
>[BestFriend] [](start = 12, length = 12)

I see no particular reason for this to be best-friended with anything. Your public surface will not involve this attribute. This attribute will only be used via entry-points and suchlike.